<ui version="4.0" stdsetdef="1" >
  <author>Stefan Schimanski &lt;1Stein@gmx.de></author>
  <class>ConfigWidget</class>
  <widget class="QWidget" name="ConfigWidget" >
    <property name="geometry" >
      <rect>
        <x>0</x>
        <y>0</y>
        <width>463</width>
        <height>376</height>
      </rect>
    </property>
    <layout class="QVBoxLayout" >
      <property name="margin" >
        <number>11</number>
      </property>
      <property name="spacing" >
        <number>6</number>
      </property>
      <item>
        <widget class="QGroupBox" name="GroupBox1" >
          <property name="title" >
            <string>Files</string>
          </property>
          <layout class="QGridLayout" >
            <property name="margin" >
              <number>11</number>
            </property>
            <property name="spacing" >
              <number>6</number>
            </property>
            <item rowspan="2" row="0" column="0" colspan="1" >
              <widget class="QListWidget" name="m_files" />
            </item>
            <item row="0" column="1" >
              <layout class="QGridLayout" >
                <property name="margin" >
                  <number>0</number>
                </property>
                <property name="spacing" >
                  <number>6</number>
                </property>
                <item row="2" column="1" >
                  <widget class="QPushButton" name="m_down" >
                    <property name="enabled" >
                      <bool>false</bool>
                    </property>
                    <property name="text" >
                      <string>&amp;Down</string>
                    </property>
                  </widget>
                </item>
                <item row="2" column="0" >
                  <widget class="QPushButton" name="m_up" >
                    <property name="enabled" >
                      <bool>false</bool>
                    </property>
                    <property name="text" >
                      <string>&amp;Up</string>
                    </property>
                  </widget>
                </item>
                <item rowspan="1" row="0" column="0" colspan="2" >
                  <widget class="QPushButton" name="m_add" >
                    <property name="text" >
                      <string>&amp;Add...</string>
                    </property>
                  </widget>
                </item>
                <item rowspan="1" row="1" column="0" colspan="2" >
                  <widget class="QPushButton" name="m_remove" >
                    <property name="enabled" >
                      <bool>false</bool>
                    </property>
                    <property name="text" >
                      <string>&amp;Remove</string>
                    </property>
                  </widget>
                </item>
              </layout>
            </item>
            <item row="1" column="1" >
              <spacer name="Spacer2" >
                <property name="sizeHint" >
                  <size>
                    <width>0</width>
                    <height>0</height>
                  </size>
                </property>
                <property name="sizeType" >
                  <enum>Expanding</enum>
                </property>
                <property name="orientation" >
                  <enum>Vertical</enum>
                </property>
              </spacer>
            </item>
          </layout>
        </widget>
      </item>
      <item>
        <widget class="QGroupBox" name="ButtonGroup1" >
          <property name="title" >
            <string>Settings</string>
          </property>
          <layout class="QGridLayout" >
            <property name="margin" >
              <number>11</number>
            </property>
            <property name="spacing" >
              <number>6</number>
            </property>
            <item rowspan="1" row="0" column="0" colspan="2" >
              <widget class="QRadioButton" name="m_single" >
                <property name="text" >
                  <string>Only show one randomly chosen medium</string>
                </property>
              </widget>
            </item>
            <item row="2" column="0" >
              <spacer name="Spacer4" >
                <property name="sizeHint" >
                  <size>
                    <width>20</width>
                    <height>21</height>
                  </size>
                </property>
                <property name="sizeType" >
                  <enum>Fixed</enum>
                </property>
                <property name="orientation" >
                  <enum>Horizontal</enum>
                </property>
              </spacer>
            </item>
            <item rowspan="1" row="1" column="0" colspan="2" >
              <widget class="QRadioButton" name="m_multiple" >
                <property name="text" >
                  <string>Switch to another medium after a delay</string>
                </property>
              </widget>
            </item>
            <item row="2" column="1" >
              <layout class="QGridLayout" >
                <property name="margin" >
                  <number>0</number>
                </property>
                <property name="spacing" >
                  <number>6</number>
                </property>
                <item row="0" column="0" >
                  <widget class="QLabel" name="m_delayLabel" >
                    <property name="enabled" >
                      <bool>false</bool>
                    </property>
                    <property name="text" >
                      <string>Delay:</string>
                    </property>
                    <property name="wordWrap" >
                      <bool>false</bool>
                    </property>
                  </widget>
                </item>
                <item row="0" column="1" >
                  <widget class="KIntSpinBox" name="m_delay" >
                    <property name="enabled" >
                      <bool>false</bool>
                    </property>
                  </widget>
                </item>
                <item rowspan="1" row="1" column="0" colspan="3" >
                  <widget class="QCheckBox" name="m_random" >
                    <property name="enabled" >
                      <bool>false</bool>
                    </property>
                    <property name="text" >
                      <string>Choose next medium randomly</string>
                    </property>
                  </widget>
                </item>
                <item row="0" column="2" >
                  <widget class="QLabel" name="m_secondsLabel" >
                    <property name="enabled" >
                      <bool>false</bool>
                    </property>
                    <property name="text" >
                      <string>seconds</string>
                    </property>
                    <property name="wordWrap" >
                      <bool>false</bool>
                    </property>
                  </widget>
                </item>
              </layout>
            </item>
            <item row="2" column="2" >
              <spacer name="Spacer16" >
                <property name="sizeHint" >
                  <size>
                    <width>0</width>
                    <height>0</height>
                  </size>
                </property>
                <property name="sizeType" >
                  <enum>Expanding</enum>
                </property>
                <property name="orientation" >
                  <enum>Horizontal</enum>
                </property>
              </spacer>
            </item>
          </layout>
        </widget>
      </item>
      <item>
        <layout class="QHBoxLayout" >
          <property name="margin" >
            <number>0</number>
          </property>
          <property name="spacing" >
            <number>6</number>
          </property>
          <item>
            <spacer name="Horizontal Spacing2" >
              <property name="sizeHint" >
                <size>
                  <width>0</width>
                  <height>0</height>
                </size>
              </property>
              <property name="sizeType" >
                <enum>Expanding</enum>
              </property>
              <property name="orientation" >
                <enum>Horizontal</enum>
              </property>
            </spacer>
          </item>
        </layout>
      </item>
    </layout>
  </widget>
  <customwidgets>
    <customwidget>
      <class>KIntSpinBox</class>
      <extends>QSpinBox</extends>
      <header>knuminput.h</header>
     </customwidget>
  </customwidgets>
    <includes>
    <include location="global" >klocale.h</include>
  </includes>
</ui>
